charts/k8s-watcher/templates/deployment.yaml

```diff
@@ -122,6 +122,10 @@ spec:
 {{- else }}
                   name: {{ include "k8s-watcher.name" . }}-secret
                   key: apiKey
+{{- end }}
+{{- if not (hasKey .Values.watcher "tags") }}
+            - name: KOMOKW_TAGS
+              value: {{ .Values.tags | default "" | quote }}
```

Comment:
Just to make sure this scenario is covered.
where or when do we set watcher.tags?
because in case it is not set you assume we have .Values.tags and it is possible we dont.

Reply:
My bad, i see the default is empty.

Reply:
It's just that its easier to set during installation a string `env:prod;team:a` rather than setting a yaml list in command line.
That's why I prefered to use this way - but still want to preserve the possibility of using a `-f values.yaml` in case customers want to use that

Reply:
So yes -- default is empty, and we recommend setting `--set tags` rather than `--set watcher.tags`